src/main/java/se/jiderhamn/promote/maven/plugin/PromoteUtils.java

```diff
@@ -129,6 +129,9 @@ static URI getTargetURI(MavenProject project) {
   static void writeProperties(Log log, MavenProject project, Properties properties) throws MojoExecutionException {
     try {
       File file = getPromotePropertiesFile(project);
+      if (file.getParentFile().mkdirs()) {
+          log.debug("Created missing target directory.");
```

Comment:
Would you mind using 2 spaces indentation to be consistent with the rest of the code?